.bower-postinstall.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+BOWER_DIR=$1
+
+if [ $BOWER_DIR ]; then
+
+  # Hopscotch - Rename .css file to .scss
```

Comment:
Why do we need to rename this to SCSS? You should either explain that in the comment, or work out a way to use the un-renamed file. (I prefer the latter, because it means we're not monkeying with the contents of things we depend on.)